api_specifications/open-api-0.1.0.yaml

```
@@ -0,0 +1,166 @@
+        '200':
+          description: ResponseInfo with case summary
+          content:
+            'application/json':
+              schema:
+                $ref: '#/components/schemas/CaseSummaryResponse'
+        '400':
+          description: querying case summary failed
+          content:
+            'application/json':
+              schema:
+                  $ref: https://raw.githubusercontent.com/egovernments/DIGIT-OSS/master/core-services/docs/common-contract_v1-1.yml#/components/schemas/ErrorRes
```

yamllint (1.29.0-1)

[error] wrong indentation: expected 16 but found 18

(indentation)